typescript/lib/constructor_calls.ts

```diff
@@ -0,0 +1,219 @@
+import { ExecCommandArgs, GenerateFilesArgs, FileGenerationOptions, LoadStaticFilesArgs, LoadLambdaArgs, GetLambdaInfoArgs, RegisterStaticFilesArgs, RegisterFilesArtifactsArgs, GetServiceInfoArgs, PartitionServices, PartitionConnections, PartitionConnectionInfo, RegisterServiceArgs, StartServiceArgs, RemoveServiceArgs, RepartitionArgs, WaitForEndpointAvailabilityArgs, ExecuteBulkCommandsArgs, ExecuteLambdaArgs } from '../kurtosis_core_rpc_api_bindings/api_container_service_pb';
+import { ServiceID } from './services/service';
+import { PartitionID } from './networks/network_context';
+import { LambdaID } from "./modules/lambda_context";
+
+// // ====================================================================================================
+// //                                    Service Context
+// // ====================================================================================================
+
+export function newExecCommandArgs(serviceId: ServiceID, command: string[]): ExecCommandArgs {
+    const result: ExecCommandArgs = new ExecCommandArgs();
+    result.setServiceId(serviceId);
+    result.setCommandArgsList(command);
+
+    return result;
+}
+
+export function newGenerateFilesArgs(serviceId: ServiceID, fileGenerationOpts: Map<string, FileGenerationOptions>): GenerateFilesArgs {
+    const result: GenerateFilesArgs = new GenerateFilesArgs();
+    result.setServiceId(String(serviceId)); 
+    for (let fileID in fileGenerationOpts) {
+        result.getFilesToGenerateMap().set(fileID, fileGenerationOpts[fileID]);
+    }
+    
+    return result;
+}
+
+export function newFileGenerationOptions(filesToGenerateSet: Set<string>): Map<string, FileGenerationOptions> {
```

Comment:
Wait this isn't right - a `newFileGenerationOptions` constructor should return a `FileGenerationOptions` object, rather than a `Map`

Reply:
See https://github.com/kurtosis-tech/kurtosis-client/blob/develop/golang/lib/binding_constructors/binding_constructors.go#L98